experts-client/queries/cdl/authorship/bind.rq

```diff
@@ -0,0 +1,25 @@
+prefix : <http://oapolicy.universityofcalifornia.edu/vocab#>
```

Comment:
I plan to add a metadata file to this, to simplify the CLI to specify a standard splay process that combines the bind, construct and frame into one argument